install.sh

```diff
@@ -3096,6 +2260,66 @@ XSD_FILES="share/doc/xsd/acct.xsd \
 
 CONTEXT_SHARE=$(find share/context/ -type f \( ! -iname "*.sh" ! -iname "SConstruct" \))
 
+#-------------------------------------------------------------------------------
+# PROMETHEUS
+#-------------------------------------------------------------------------------
+
+ALERTMANAGER_VENDOR_DIR='alertmanager'
+NODE_EXPORTER_VENDOR_DIR='node_exporter'
+PROMETHEUS_VENDOR_DIR='prometheus'
+if [ $ARCH = 'arm64' ]; then
+    ALERTMANAGER_VENDOR_DIR='alertmanager.arm64'
+    NODE_EXPORTER_VENDOR_DIR='node_exporter.arm64'
+    PROMETHEUS_VENDOR_DIR='prometheus.arm64'
+
+    # adjust restic binary symlink
+    rm src/datastore_mad/remotes/restic/restic
+    ln -s ./vendor/bin/restic.arm64 src/datastore_mad/remotes/restic/restic
+fi
+
+# ALERTMANAGER
+ONEPROMETHEUS_ALERTMANAGER_BIN_FILES="src/oneprometheus/vendor/${ALERTMANAGER_VENDOR_DIR}/alertmanager \
+                                      src/oneprometheus/vendor/${ALERTMANAGER_VENDOR_DIR}/amtool"
+ONEPROMETHEUS_ALERTMANAGER_CONFIG_FILES="src/oneprometheus/alertmanager/etc/alertmanager.yml"
+ONEPROMETHEUS_ALERTMANAGER_FILES="src/oneprometheus/vendor/${ALERTMANAGER_VENDOR_DIR}/LICENSE \
+                                  src/oneprometheus/vendor/${ALERTMANAGER_VENDOR_DIR}/NOTICE"
+ONEPROMETHEUS_ALERTMANAGER_SYSTEMD_FILES="src/oneprometheus/alertmanager/systemd/opennebula-alertmanager.service"
+
+# GRAFANA
+ONEPROMETHEUS_GRAFANA_FILES="src/oneprometheus/grafana/share/dashboards/"
+
+# LIBVIRT-EXPORTER
+ONEPROMETHEUS_LIBVIRT_EXPORTER_FILES="src/oneprometheus/opennebula-libvirt-exporter/src/libvirt_collector.rb \
+                                      src/oneprometheus/opennebula-libvirt-exporter/src/libvirt_exporter.rb"
+ONEPROMETHEUS_LIBVIRT_EXPORTER_SYSTEMD_FILES="src/oneprometheus/opennebula-libvirt-exporter/systemd/opennebula-libvirt-exporter.service"
+
+# NODE-EXPORTER
+ONEPROMETHEUS_NODE_EXPORTER_BIN_FILES="src/oneprometheus/vendor/${NODE_EXPORTER_VENDOR_DIR}/node_exporter"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ONEPROMETHEUS_NODE_EXPORTER_BIN_FILES appears unused. Verify use (or export if used externally).